### ISSUE TITLE
XML validation against XSD schema caused wrong error after comment

### ISSUE THREAD

**Issue by HofmannMario** [redhat-developer/vscode-xml#929]
If XML document contains a comment the second tag after the comment is marked by red line
showing error 'XML: More than one root element'.
![VSCode-Error (1)](https://github.com/redhat-developer/vscode-xml/assets/143386732/032d8e54-0406-437a-a07b-63f861fd1292)
![VSCode-Error (2)](https://github.com/redhat-developer/vscode-xml/assets/143386732/2b6e06d3-2998-45cf-80bc-5f467471ce7e)

If comment line is removed the error is gone:
![VSCode-Error (3)](https://github.com/redhat-developer/vscode-xml/assets/143386732/f05c5bc4-33b4-4071-8ccb-10a6570ab8bf)

**Comment by angelozerr** [comment]
Thats very strange since validation is managed with xerces.

Could you share your xml please

**Comment by HofmannMario** [comment]
Not really, it's a business source.

It seems the problem can reproduced with any simply ant build file (as the caused file also is). Hier is a similar file also showing the error:
[projectfile.zip](https://github.com/redhat-developer/vscode-xml/files/12452786/projectfile.zip)

**Comment by fbricon** [comment]
@HofmannMario try disabling all extensions but vscode-xml, see if you still see this issue

**Comment by angelozerr** [comment]
@HofmannMario I tested your XML and vscode-xml doesn't report your error. I suspect your error comes from an another vscode extension.

More, vscode-xml report error by given an error code and your message doesn't provide it is an another reason why I think it is an another vscode extension which reports this error. By searching quickly, I have found https://forums.codeguru.com/showthread.php?245766-Reading-an-XML-file-with-multiple-root-elements which is an XML parser written in CSharp, so I suspect that you have installed a vscode extension which validates XML with this CSharp parser (not sure for that).
